state/protocol/badger/state.go

```diff
@@ -188,27 +191,23 @@ func Bootstrap(
 		if err != nil {
 			return fmt.Errorf("could not bootstrap version beacon: %w", err)
 		}
-
-		err = updateEpochMetrics(metrics, root)
-		if err != nil {
-			return fmt.Errorf("could not update epoch metrics: %w", err)
-		}
-		metrics.BlockSealed(lastSealed)
-		metrics.SealedHeight(lastSealed.Header.Height)
-		metrics.FinalizedHeight(lastFinalized.Header.Height)
-		for _, block := range segment.Blocks {
-			metrics.BlockFinalized(block)
-		}
-
```

Comment:
These lines here together with [lines 220 - 245 of _the old code_](https://github.com/onflow/flow-go/blob/c0f316d67e1701db1f04240f7fcf115890e89987/state/protocol/badger/state.go#L220-L245) essentially is the functionality of `OpenState`. Hence now just call `OpenState` below and therefore can remove this code. 

In addition, the usage of `OpenState` serves as a nice sanity check that the result of the bootstrapping process is actually accepted by `OpenState`.